### PR TITLE
JavaScript capstone project

### PR DESCRIPTION
**Pull request by RayhanTabase**
## JavaScript capstone project
To complete this capstone we:
- Worked with the AmiiboAPI to receive a list of characters
-  Involvement API to record the different user interactions (likes, comments)
- Built the home page.
- Built the comments popup.
- Built counter functions for items and comments
- Used webpack
- Used jest for testing
- Used git-flow
- Used pair programming
- [Link to video](https://drive.google.com/file/d/1uWRGgGf2-dIB1MxPkByqx1PbaKrkF1-2/view?usp=sharing)
- Video : [What made us proud](https://drive.google.com/file/d/1EhtGs2Xej7iMn3xOGBeXeGutCy2axJI1/view?usp=sharing)

### CODE REVIEW

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @RayhanTabase && @charlesgobina, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] The project requires that all team members participate in the project presentation. Kindly record another video presentation and make sure that all team members participate.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi there, 

Awesome job on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Let's work on those.


 ## Required Changes ♻️
_Kindly check the comments under the review._


Happy coding, Cheers!🚀🚀

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,135 @@
 # Pokedex
-A simple web app that shows details concerning different Pokémon. Built using HTML,CSS, and JavaScript
+A simple web app that shows details concerning different Amiibo. Built using HTML,CSS, and JavaScript. Users view a list of amiibo and have the possibility to like and comment on their favorite amiibo.
+
+## Video presentation
+* [Video](https://drive.google.com/file/d/1uWRGgGf2-dIB1MxPkByqx1PbaKrkF1-2/view?usp=sharing)
```
Comment (mikenath223) [inline README.md:5]: - [ ] Kindly record an additional video highlighting a piece of code that you found interesting or are proud of making.

**Review by addod19 — APPROVED** [review]
### Hi @RayhanTabase and @charlesgobina, 

Your project is complete!  Great job both of you guys did a great presentation. Keep up the awesome job :fireworks: 
There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**